runtime/errors.go

```diff
@@ -81,6 +81,21 @@ func HTTPError(ctx context.Context, mux *ServeMux, marshaler Marshaler, w http.R
 	mux.errorHandler(ctx, mux, marshaler, w, r, err)
 }
 
+// HttpStreamError uses the mux-configured stream error handler to notify error to the client without closing the connection.
+func HttpStreamError(ctx context.Context, mux *ServeMux, marshaler Marshaler, w http.ResponseWriter, r *http.Request, err error) {
```

Comment:
Could we name this 
```suggestion
func HTTPStreamError(ctx context.Context, mux *ServeMux, marshaler Marshaler, w http.ResponseWriter, r *http.Request, err error) {
```
For consistency? I know it's not the recommended pattern but having the two next to each other in the public API would be cursed.

Reply:
For sure. I forgot that this is actually the recommended pattern for Golang